apps/meteor/ee/client/voip/modal/DialPad/hooks/useDialPad.tsx

```diff
@@ -41,18 +47,33 @@ export const useDialPad = ({ initialValue, errorMessage }: DialPadProps): DialPa
 	});
 
 	const { ref, onChange } = register('PhoneInput');
+
 	const value = watch('PhoneInput');
 
 	const [disabled, setDisabled] = useState(true);
 
 	const handleBackspaceClick = useCallback((): void => {
+		clearErrors();
 		setValue('PhoneInput', value.slice(0, -1));
-	}, [setValue, value]);
+	}, [clearErrors, setValue, value]);
 
 	const handlePadButtonClick = useCallback(
 		(digit: string | number): void => {
+			clearErrors();
+
 			setValue('PhoneInput', value + digit);
 		},
+		[clearErrors, setValue, value],
+	);
+
+	const handlePadButtonLongPressed = useCallback(
+		(digit: string | number): void => {
+			if (digit === 0) {
+				return setValue('PhoneInput', `${value}+`);
+			}
+
+			return setValue('PhoneInput', value + digit);
```

Comment:
Won't long pressing any digit other than 0 add something like `ABC`? It doesn't seem very useful.

Reply:
I was developing same behavior that we found on competitors dial pad... only zero is important to us as a long press at this point.